src/Pages/Checkout/Checkout.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import Shipping from './Component/Shipping/Shipping';
+import Payment from './Component/Payment/Payment';
+import { BASE_URL } from '../../config.js';
+import { Token } from '../../config.js';
+import Order from './Component/Order/Order';
+import './Checkout.scss';
+
+//부모:마이박스에서 섭토탈, 토탈 받아오기
+const Checkout = ({ subtotal, total }) => {
+  const [orderInfo, setOrderInfo] = useState({
+    zipCode: null,
+    addressMain: null,
+    addressDetail: null,
+    updateAddress: null,
+    wayOfShipping: null,
+    wayOfPayment: null,
+  });
+
+  const orderInfoHandler = e => {
+    setOrderInfo({ ...orderInfo, [e.target.className]: e.target.value });
+  };
```

Comment:
```suggestion
  const orderInfoHandler = e => {
    const { className, value } = e.target;
    setOrderInfo({ ...orderInfo, [className]: value });
  };
```
구조분해 할당하면 더 깔끔하겠네요!
하지만 이런 부분들에서input의 className이나 Id 값을 사용하기 보다는 name 속성을 잘 이용하는게 훨씬 깔끔합니다.

Reply:
반영하였습니다.